frontend/src/app/(dashboard)/articles/[id]/page.tsx

```diff
@@ -0,0 +1,16 @@
+import { ArticleProps } from "@/types/Article";
+import React from "react";
+
+export default async function ArticlePage({
+  params,
+}: {
+  params: { id: string };
+}) {
+  const article: ArticleProps = await fetch(
+    `${process.env.API_ORIGIN}/articles/${params.id}`
```

Comment:
Need to change to `${process.env.API_ORIGIN}/article/${params.id}` or change the backend endpoint to `/articles/:id`.